docs/usage/lodge_support_ticket.md

```diff
@@ -0,0 +1,16 @@
+## I need help using cloud.gov.au
+
+We use Zendesk to manage support tickets for cloud.gov.au. This service is available for anyone with a .gov.au email address.
+
+Before you lodge a support ticket be sure to check our documentation. You can also check the [status of cloud.gov.au](http://status.cloud.gov.au/) if you are experiencing an outage.
+
+<a href="https://digital89.zendesk.com/" role="button">Lodge a support ticket</a>
```

Comment:
Testing with someone who has a zendesk account but isn't an agent, going to this address they had to:

1. Sign in (can sign in with google)
2. Find the submit feedback link (hiding in the top bar)

It's probably fine for now but might require improving later.